lib/api.js

```diff
@@ -70,42 +70,31 @@ class API {
   post(params, cb) {
      let request = {
         url: this.getEntityUrl(params),
-        form: params.data
+        body: params.data
```

Comment:
From the `request-promise` documentation, we add `json:true` to post a json payload. Don't we need that here?
https://www.npmjs.com/package/request-promise#post-data-to-a-json-rest-api

Reply:
We are already sending a payload in json format so i dont think we need to add this `json:true`. Please correct if i am wrong

Reply:
Can you run the test case once with `json:true` and check? 
Also, when we get the response, do we get it as parsed JSON or as a string?

Reply:
Looks like `json:true` is added as default option in `request.defaults`